src/utils/updates.ts

```diff
@@ -1,148 +0,0 @@
-import { iterateLatest, makeFollower, makeLeader } from '@agoric/casting';
```

Comment:
I guess this is just because we hard forked dapp-PSM

Reply:
exactly. then started getting data directly from storage paths with hooks